python/mlc_llm/bench/executor.py

```diff
@@ -147,9 +153,147 @@ def get_executor_feature_dict(self) -> Dict[str, Any]:
         return {"num_concurrent_requests": self.num_concurrent_requests}
 
 
-# Todo: Timestamp executor for fixed request rate or log replay  # pylint: disable=fixme
-# class FixTimestampExecutor(Executor):
-#     pass
+class FixTimestampExecutor(Executor):
+    """The benchmark executor of fixing the timestamps of sending requests."""
+
+    launch_start_gap_time: float = 1.0
+
+    def __init__(  # pylint: disable=too-many-arguments
+        self,
+        f_create_api_endpoint: Callable[[], APIEndPoint],
+        num_processes: Optional[int],
+        disable_tqdm: bool,
+        num_requests: int,
+        request_rate: Optional[np.float32] = None,
```

Comment:
remove request_rate from executor, instead, construct a different pipeline when request_rate is needed